pelican/tests/test_utils.py

```diff
@@ -78,13 +81,13 @@ def test_get_date(self):
         iso_8601_date = utils.SafeDatetime(year=1997, month=7, day=16)
         iso_8601_date_hour_tz = utils.SafeDatetime(
             year=1997, month=7, day=16, hour=19, minute=20,
-            tzinfo=pytz.timezone('CET'))
+            tzinfo=zoneinfo.ZoneInfo('CET'))
         iso_8601_date_hour_sec_tz = utils.SafeDatetime(
             year=1997, month=7, day=16, hour=19, minute=20, second=30,
-            tzinfo=pytz.timezone('CET'))
+            tzinfo=zoneinfo.ZoneInfo('CET'))
         iso_8601_date_hour_sec_ms_tz = utils.SafeDatetime(
             year=1997, month=7, day=16, hour=19, minute=20, second=30,
-            microsecond=450000, tzinfo=pytz.timezone('CET'))
+            microsecond=450000, tzinfo=zoneinfo.ZoneInfo('CET'))
         iso_8601 = {
             '1997-07-16': iso_8601_date,
             '1997-07-16T19:20+01:00': iso_8601_date_hour_tz,
```

Comment:
Maybe it's better to move these dates to regular CET instead, e.g. December (`1997-12-16T...`). It might be confusing to see `CET` and `+2` at first glance.

Reply:
That makes sensse, although the tests will still fail since we are setting the tzinfo to CET but when we are comparing the values, we don't have the tzinfo, so the expected isn't the same as the value.

I believe the +1 was just a way to maybe set the tz automatically?

The odd thing here is that I testing this with pytz and the test should have failed since the tzinfo isnt being set either 🤔

Reply:
Timezones and UTC offsets are more or less interchangeable. In other words, a datetime with a timezone set should compare equal to an ISO date with appropriate UTC offset (`+01:00` etc).

Reply:
Main difference is that timezones are dynamic and ISO offset is static. And by dynamic, I mean stuff like summer time or countries/timezones shift offsets due to government decisions over time, etc. So a timezone would calculate (or [more like lookup from a database](https://en.wikipedia.org/wiki/Tz_database#Example_zone_and_rule_lines)) the correct offset based on the date.

Reply:
That makes sense, the issue with the test is that the check of equality fails. It's a bit odd that with pytz the test is passing since it returns the same value that TZ=(None, 3600)

Reply:
Yes, and that was wrong. It should be `7200` sec, because it's summer time for CET for those dates. It looks like a bug in `pytz` and tests seem to be written with that bug in place. That's why I said the tests should say `+02` or we should move out of the summer time and keep `+01` (i.e. December).